qemu/qemu_test.go

```diff
@@ -1026,6 +1027,27 @@ func TestBadGlobalParam(t *testing.T) {
 	}
 }
 
+func TestBadPFlash(t *testing.T) {
+	c := &Config{}
+	c.appendPFlashParam()
+	if len(c.qemuParams) != 0 {
+		t.Errorf("Expected empty qemuParams, found %s", c.qemuParams)
+	}
+}
+
+func TestValidPFlash(t *testing.T) {
+	c := &Config{}
+	Expected := []string{"flash0", "flash1"}
```

Comment:
Micro nit: this should be `expected` really as leading capital variable names have significance in golang. The linters might complain about this. Also, the name is slightly misleading as this isn't the expected value - the refefinition of `Expected` below (which should also be `expected`) is :smile: